roles/benchmarking_run_nvidiadl_ssd/tasks/main.yml

```diff
@@ -77,3 +77,15 @@
       oc describe pod/{{ benchmarking_nvidiadl_ssd_name }} -n {{ benchmarking_namespace }}
         > "{{ artifact_extra_logs_dir }}/pod_run-nvidiadl-ssd.descr"
     failed_when: false
+
+- name: Get average sample rate
+  shell:
+    set -o pipefail;
+    cat "{{ artifact_extra_logs_dir }}/pod_run-nvidiadl-ssd.log" | grep avg. | tail -n1 | awk '{ print $NF }' > "{{ artifact_dir }}/benchmarking_run_ssd_sample_rate.log";
```

Comment:
can you add a unit here, so that the file content is easier to read? (mind that the asses function should be updated as well)
```
awk '{ print $NF " units" }'
```